Update recurring action workflow to account for uptodate state changes (#3184)
uyuni-project/uyuni-docs: modules/common-workflows/pages/workflow-clients-update-rec-actions.adoc

  For example, ** Weekly: Wednesday, 9:30 am **.
  Keep enough time between this action and the [literal]``update-salt`` action.
  The [literal]``update-salt`` actions must be finished on all systems before this action should be executed.

> . Assign the states [literal]``util.syncall``, [literal]``certs``, [literal]``channels`` and [literal]``uptodate`` by selecting the checkboxes. If a reboot is desired you can also add [literal]``reboot`` or [literal]``rebootifneeded``.

Reviewer: ```suggestion
. Assign the states [literal]``util.syncall``, [literal]``certs``, [literal]``channels`` and [literal]``uptodate`` by selecting the checkboxes.
  To perform a reboot you can also add [literal]``reboot`` or [literal]``rebootifneeded``.
```
